src/globus_sdk/_testing/data/timer/_common.py

```diff
@@ -0,0 +1,157 @@
+import uuid
+
+TIMER_ID = str(uuid.uuid1())
+
+DEST_EP_ID = str(uuid.uuid1())
+SOURCE_EP_ID = str(uuid.uuid1())
+
+
+V2_TRANSFER_TIMER = {
+    "body": {
+        "DATA": [
+            {
+                "DATA_TYPE": "transfer_item",
+                "destination_path": "/~/dst.txt",
+                "source_path": "/share/godata/file1.txt",
+            }
+        ],
+        "DATA_TYPE": "transfer",
+        "delete_destination_extra": False,
+        "destination_endpoint": DEST_EP_ID,
+        "encrypt_data": False,
+        "fail_on_quota_errors": False,
+        "notify_on_failed": True,
+        "notify_on_inactive": True,
+        "notify_on_succeeded": True,
+        "preserve_timestamp": False,
+        "skip_source_errors": False,
+        "source_endpoint": SOURCE_EP_ID,
+        "store_base_path_info": False,
+        "verify_checksum": True,
+    },
+    "inactive_reason": None,
+    "job_id": TIMER_ID,
+    "last_ran_at": None,
+    "name": "Very Cool Timer",
+    "next_run": "2023-10-27T05:00:00+00:00",
+    "number_of_errors": 0,
+    "number_of_runs": 0,
+    "schedule": {
+        "end": {"count": 2},
+        "interval_seconds": 604800,
+        "start": "2023-10-27T05:00:00+00:00",
+    },
```

Comment:
```suggestion
    "schedule": {
        "type": "recurring",
        "end": {"count": 2},
        "interval_seconds": 604800,
        "start": "2023-10-27T05:00:00+00:00",
    },
```

Reply:
:facepalm: Yep, not even valid in my version. :+1: